### ISSUE TITLE
Unable to set hyperparams

### ISSUE THREAD

**Issue by RutujaWanjari** [OliverEdholm/Convolutional-Autoencoder#1]
Hello Oliver,

Thanks for such a great repo. I am trying to follow it.
I have changed some hyperparameters like -
1. Dataset - 35,000 (200x200 RGB images of cropped faces with variance in gender, age, race, lighting conditions, etc)
2. epoch - started from 8 to 40
3. learning rate - 0.05 (tried 0.1, 0.005)

The problem is, after few epochs the model starts fluctuating at loss of (0.49 to 0.51) or (0.52 to 0.53).
I wanted to know whether I am doing it correctly.

Last try was with - 32 epochs, 0.05 learning rate, 35k cropped face images, batch size 16.
Result - Even after 25 epochs loss was fluctuating at 0.55 - 0.54

Thanks in advance.

**Comment by RutujaWanjari** [comment]
Ok I got the issue. Even though loss is fluctuating, I need to carry on training. The models created gives very good result. Previously, I was stopping training if I see fluctuating results for some time.
